File: ProyectoFinalVB/ProyectoFinalVB/Menu.vb
Public Class Menu
    Private Sub btnAlgoritmo_Click(sender As Object, e As EventArgs) Handles btnAlgoritmo.Click
        Dim abrir As MenuAlgoritmos = New MenuAlgoritmos()
        abrir.Show()
    End Sub

    Private Sub btnEstructura_Click(sender As Object, e As EventArgs) Handles btnEstructura.Click
        Dim aperto As MenuEstructuras = New MenuEstructuras()
        aperto.Show()
    End Sub
End Class

File: ProyectoFinal/ProyectoFinal/Menu.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace ProyectoFinal
{
    public partial class Menu : Form
    {
        public Menu()
        {
            InitializeComponent();
        }
       
        private void btnAlgoritmo_Click(object sender, EventArgs e)
        {
            MenuAlgoritmos abrir = new MenuAlgoritmos();
            abrir.Show();
            Hide();
        }

        private void btnEstructura_Click(object sender, EventArgs e)
        {
            MenuEstructuras aperto = new MenuEstructuras();
            aperto.Show();
            Hide();
        }
    }
}
